R/flywire-skeleton.R

```diff
@@ -359,17 +364,23 @@ py_skeletor <- function(id,
   if(reroot){
     neuron = reroot_hairball(neuron, k.soma.search = k.soma.search, radius.soma.search = radius.soma.search, brain = brain)
   }
-  if(mesh3d){
+  if(mesh3d|save.obj){
     if(is.null(mesh)){
       # we need to get python to export it
-      savedir <- tempdir()
+      savedir <- if(!is.null(save.obj)){
+        save.obj
+      }else{
+        tempdir()
```

Comment:
`tempdir()` is the root temporary folder of the R session. You should not delete it (as you found out the other day when you were deleting bridging registrations). It is better to make your own temporary folder

```
dir.create(td<-tempfile())
td
```

Reply:
@alexanderbates: you should also add an on.exit hook to delete that folder on exit
```
on.exit(unlink(td, recursive=TRUE))
```

Reply:
Okay, done